src/WebHandlerImpl.h

```diff
@@ -55,7 +56,7 @@ class AsyncStaticWebHandler: public AsyncWebHandler {
     AsyncStaticWebHandler& setDefaultFile(const char* filename);
     AsyncStaticWebHandler& setCacheControl(const char* cache_control);
     AsyncStaticWebHandler& setLastModified(const char* last_modified);
-    AsyncStaticWebHandler& setLastModified(struct tm* last_modified);
+    AsyncStaticWebHandler& setLastModified(const std::tm* last_modified);
```

Comment:
is this change backward compatible ? or it should be a second function instead ?

Reply:
I want to keep a full backward compatibility with the existing API in this fork. That's the number 1 priority. So if it can break an existing code, it has to be renamed or overloaded.

Reply:
OK, I'll add overloaded method also then

Reply:
which means you could simplify it in `AsyncStaticWebHandler& setLastModified(const tm* last_modified);` ? No need for `struct` tor `std::` ?